packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -292,7 +300,7 @@ const getS3Buckets = async (account: AWSAccountInfo): Promise<S3BucketInfo[]> =>
         });
       }
     } catch (e) {
-      if (e.code !== 'NoSuchTagSet' && e.code !== 'NoSuchBucket') {
+      if (e.code !== 'NoSuchTagSet' && e.code !== 'NoSuchBucket' && e.code !== 'IllegalLocationConstraintException') {
```

Comment:
Please see https://github.com/aws-amplify/amplify-category-api/pull/2269

The `IllegalLocationConstraintException` appears to be related to buckets created in opt-in regions. The proper fix for that appears to be to use a regionalized S3 client to get the bucket tagging. However, as noted in that PR, there are some buckets that were created in opt-in regions other than the one to which the account itself is opted-in! We don't quite understand how that happened (maybe it was previously opted into that region?) but those buckets can't be operated on.